### PR TITLE
Speech SDK Update

### PR DESCRIPTION
**Pull request by olmidy**
### Purpose
-Update Speech SDK to v1.10.0
-Add support for Custom Commands

### Changes
*Are there any changes that need to be called out as significant or particularly difficult to grasp? (Include illustrative screenshots for context if applicable.)*

### Tests
*Is this covered by existing tests or new ones? If no, why not?*

### Feature Plan
*Are there any remaining steps or dependencies before this issue can be fully resolved? If so, describe and link to any relevant pull requests or issues.*

#### General
- [x ] I have commented my code, particularly in hard-to-understand areas	
- [x ] I have added or updated the appropriate tests	
- [x ] I have updated related documentation

### CODE REVIEW

File: samples/android/clients/VirtualAssistantClient/directlinespeech/src/main/java/com/microsoft/bot/builder/solutions/directlinespeech/ConfigurationManager.java [hunk 1]
```diff
@@ -52,6 +52,9 @@ public Configuration getConfiguration(){
         if (configuration.serviceKey == null) {
             configuration.serviceKey = defaultConfiguration.serviceKey;
         }
+        if (configuration.customCommandsAppId == null) {
```
Comment (ryanisgrig) [inline samples/android/clients/VirtualAssistantClient/directlinespeech/src/main/java/com/microsoft/bot/builder/solutions/directlinespeech/ConfigurationManager.java:59]: How is the Custom Commands app id picked up in the SDK?

**Review by ryanisgrig — CHANGES_REQUESTED** [review]
1. Update the [default configuration JSON](https://github.com/microsoft/botframework-solutions/blob/9f2cbd73b1b5ced1d463ce39122994d771c49c2c/samples/android/clients/VirtualAssistantClient/directlinespeech/src/main/assets/default_configuration.json) adding `commands_app_id` property.
1. Doc update giving a quick blurb/link to Custom Commands doc and showing JSON change at [/docs/_docs/clients-and-channels/clients/virtual-assistant-client.md](https://github.com/microsoft/botframework-solutions/blob/9f2cbd73b1b5ced1d463ce39122994d771c49c2c/docs/_docs/clients-and-channels/clients/virtual-assistant-client.md)

**Comment by ryanisgrig** [comment]
I have an issue debugging locally
```
2020-03-19 16:41:29.629 20504-20504/? A/DEBUG: *** *** *** *** *** *** *** *** *** *** *** *** *** *** *** ***
2020-03-19 16:41:29.629 20504-20504/? A/DEBUG: Build fingerprint: 'google/sdk_gphone_x86/generic_x86:10/QSR1.191030.002/5978551:userdebug/dev-keys'
2020-03-19 16:41:29.629 20504-20504/? A/DEBUG: Revision: '0'
2020-03-19 16:41:29.629 20504-20504/? A/DEBUG: ABI: 'x86'
2020-03-19 16:41:29.630 20504-20504/? A/DEBUG: Timestamp: 2020-03-19 16:41:29-0700
2020-03-19 16:41:29.630 20504-20504/? A/DEBUG: pid: 20466, tid: 20466, name: irtualassistant  >>> com.microsoft.bot.builder.solutions.virtualassistant <<<
2020-03-19 16:41:29.630 20504-20504/? A/DEBUG: uid: 10133
2020-03-19 16:41:29.630 20504-20504/? A/DEBUG: signal 11 (SIGSEGV), code 1 (SEGV_MAPERR), fault addr 0x1c
2020-03-19 16:41:29.630 20504-20504/? A/DEBUG: Cause: null pointer dereference
2020-03-19 16:41:29.631 20504-20504/? A/DEBUG:     eax 00000000  ebx c492eaf0  ecx 0000001c  edx 0000001c
2020-03-19 16:41:29.631 20504-20504/? A/DEBUG:     edi e49cae7c  esi 0000001c
2020-03-19 16:41:29.631 20504-20504/? A/DEBUG:     ebp d932a1c0  esp ffe17484  eip eee50a62
2020-03-19 16:41:29.684 20504-20504/? A/DEBUG: backtrace:
2020-03-19 16:41:29.684 20504-20504/? A/DEBUG:       #00 pc 00090a62  /apex/com.android.runtime/lib/bionic/libc.so (strlen_generic+18) (BuildId: 76290498408016ad14f4b98c3ab6c65c)
2020-03-19 16:41:29.684 20504-20504/? A/DEBUG:       #01 pc 000a36db  /data/app/com.microsoft.bot.builder.solutions.virtualassistant-Ne3V7dwEqgJayBfsNW6dRw==/lib/x86/libMicrosoft.CognitiveServices.Speech.java.bindings.so
2020-03-19 16:41:29.684 20504-20504/? A/DEBUG:       #02 pc 0007613f  [anon:libc_malloc]
2020-03-19 16:41:29.856 1869-1869/? E//system/bin/tombstoned: Tombstone written to: /data/tombstones/tombstone_09
```

**Comment by darrenj** [comment]
Thanks @olmidy. @ryanlengel could you validate the change with our VA?

**Review by ryanisgrig — APPROVED** [review]
Thanks @olmidy! Validated with my Android device and VA environment.

**Review by darrenj — APPROVED** [review]

**Pull request merged**